src/main/java/frc/robot/subsystems/HopperSubsystem.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+
+/** Add your docs here. */
+public class HopperSubsystem extends SubsystemBase {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+  private CANSparkMax m_motorController3 = new CANSparkMax(Constants.kMotorThreeID, CANSparkMaxLowLevel.MotorType.kBrushless);
+  
+  public HopperSubsystem(boolean on){
```

Comment:
This is your constructor, it is called just once when the HopperSubsystem is created (using "new"). It is used to assign values to any of the object's data and potentially set an initial state. If you want a way to control whether the Hopper is running at a given time, you should create a new method for that purpose.